### PR TITLE
Update/project links

### PR DESCRIPTION
**Pull request by shaunmulligan**
Update some links in seed projects to point at new resin-io-projects. Also add 2 projects to the projects page.

### CODE REVIEW

File: pages/examples/projects.md [hunk 1]
```diff
@@ -27,6 +27,20 @@ These are some awesome projects built using resin.io and are a lot more fully fe
...
+
+This is a simple node.js project built on resin.io. It should work on all versions of the Raspberry Pi.
+
+The audio stock ticker will verbally announce a list of your favourite stocks every couple of minutes or hours, depending on how you configure it.
+
+### Process and Device Health Monitor
+
+[Repository][pm2-repo-link]
+
+This is a simple example project for Resin.io which demonstrates how to use pm2 on your device to keep a process running forever and monitor your devices health. For the full story check our [blog post][pm2-blog-link] about it.
```
Comment (craig-mulligan) [inline pages/examples/projects.md]: What happened to lower case resin.io?

File: pages/examples/seed-projects.md [hunk 2]
```diff
@@ -37,12 +37,21 @@ This is a simple Hello, World project for [node.js][node] designed to act as a
 basis for future work. It demonstrates how to install native Linux packages and
 configure your application.
 
+[Repository][simple-nodejs-server-link]
+
+This seed project runs a simple [Express.js][expressjs-link] server on port `:80`.
+It is slightly different from the simple-nodejs project as it includes a `Dockerfile`, this allows for more flexibility at build time and better caching of builds.
```
Comment (craig-mulligan) [inline pages/examples/seed-projects.md]: This could link to the Dockerfile guide?

**Comment by craig-mulligan** [comment]
Other than those few comments. LGTM

File: pages/examples/projects.md (hunk 1 above)
Comment (shaunmulligan) [inline pages/examples/projects.md]: haha, I copied that directly from your description on the repo:
https://github.com/resin-io-projects/resin-pm2-monitor
I will fix it now. thanks

Comment (craig-mulligan) [inline pages/examples/projects.md]: Great, I've fixed it in the resin-pm2-monitor readme

**Pull request merged**